addons/website_forum/models/forum.py

```diff
@@ -141,7 +142,7 @@ def _compute_forum_statistics(self):
             result[cid]['total_posts'] += res_group.get('__count', 0)
             result[cid]['total_views'] += res_group.get('views', 0)
             result[cid]['total_answers'] += res_group.get('child_count', 0)
-            result[cid]['total_favorites'] += res_group.get('favourite_count', 0)
+            result[cid]['total_favorites'] += 1 if res_group.get('favourite_count', 0) else 0
```

Comment:
just to be sure to understand this change... whats your purpose?

Reply:
SBU asked him that in the chatter of the task ;) "For the Favorite stat buttons on a forum form view, display the number of posts that have been marked as favorite (not the total of favorites). With the aggregate (sum) on the columns of the tree view, it'll help."